### PR TITLE
Updated The Readme file

### PR DESCRIPTION
**Pull request by aviavinashkr**
- Added Heading for different sections
-Added attractive Emoji's

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,37 +12,33 @@ The main functionalities
 
 - Displays all the important laws that a consumer must be aware of
 
+# Screenshots
 
-<hr>  
-
+## Login & Register Page. 📟
```
Comment (annu12340) [inline README.md]: Hey the idea of adding emojis is good.
Could you add different relevant emoji for each heading
Change the heading size ```from ## Login & Register Page. to #### Login & Register Page.```

**Review by annu12340 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aviavinashkr) [reply README.md]: Yeah sure I can do it.

**Comment by annu12340** [comment]
Pls resolve the conflicts

**Comment by aviavinashkr** [comment]
I Have added the changes that you asked for.

**Comment by annu12340** [comment]
Thanks for the pr 😄

**Comment by aviavinashkr** [comment]
No Problem

**Pull request merged**